data/countries_names.txt

```diff
@@ -10072,7 +10072,7 @@
   da = Ille-et-Vilaine
   nl = Ille-et-Vilaine
   fi = Ille-et-Vilaine
-  fr = Ille-et-Vilaine‎
+  fr = Ille-et-Vilaine
```

Comment:
What has changed here and in other similar places? Line ending? Can it be avoided?

Reply:
I have remove invisible spaces  in line ending